### PR TITLE
Shaokai/0.1.3

### PR DESCRIPTION
**Pull request by yeshaokai**
Fixed a bug

Changed hardcoded paths to relative paths in demo notebooks

### CODE REVIEW

**Comment by MMathisLab** [comment]
so this would be 0.1.3? or should we wait a bit to catch bugs? :D

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ In our original work (NeurIPS 2023) we used GPT3.5 and GPT4 as part of our agent
 Conda is an easy-to-use Python interface that supports launching [Jupyter Notebooks](https://jupyter.org/). If you are completely new to this, we recommend checking out the [docs here for getting conda installed](https://deeplabcut.github.io/DeepLabCut/docs/beginner-guides/beginners-guide.html#beginner-user-guide). Otherwise, proceed to use one of [our supplied conda files](https://github.com/AdaptiveMotorControlLab/AmadeusGPT/tree/main/conda). As you will see we have minimal dependencies to get started, and [here is a simple step-by-step guide](https://deeplabcut.github.io/DeepLabCut/docs/installation.html#step-2-build-an-env-using-our-conda-file) you can reference for setting it up (or see [BONUS](README.md#bonus---customized-your-conda-env) below). Here is the quick start command:
 
 ```bash
-conda env create -f amadeusGPT.yml
+conda env create -f conda/amadeusGPT.yml
```
Comment (MMathisLab) [inline README.md:39]: please drop conda; that is not useful is one downloads the file, for example.

File: notebooks/EPM_demo.ipynb [hunk 2]
```diff
@@ -174,9 +174,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "amadeusgpt-minimal",
+   "display_name": "amadeusgpt-cpu",
    "language": "python",
-   "name": "python3"
+   "name": "amadeusgpt-cpu"
```
Comment (MMathisLab) [inline notebooks/EPM_demo.ipynb:179]: drop -cpu

**Review by MMathisLab — CHANGES_REQUESTED** [review]

File: notebooks/EPM_demo.ipynb (hunk 2 above)
Comment (yeshaokai) [reply notebooks/EPM_demo.ipynb:179]: This is just a meta info that stores my conda environment name. I don't know why it's there as I hit clear all outputs

File: README.md (hunk 1 above)
Comment (yeshaokai) [reply README.md:39]: That's the PR from Alex though.  Marc also told me he was confused why there was no conda/.

Comment (MMathisLab) [reply README.md:39]: well, if one is within the conda/ folder this cannot be run either, so it's best to keep it not there and use that Q as an opportunity to educate :D

**Comment by yeshaokai** [comment]
@MMathisLab  Some improvements based on how residents used it. This can be 0.1.3.

**Review by MMathisLab — APPROVED** [review]

**Pull request merged**